go/tools/builders/generate_test_main.go

```diff
@@ -191,14 +191,15 @@ func main() {
 	if failfast := os.Getenv("TESTBRIDGE_TEST_RUNNER_FAIL_FAST"); failfast != "" {
 		flag.Lookup("test.failfast").Value.Set("true")
 	}
-
-	// Setting this flag serves two purposes:
-	// 1. It attains parity with "go test", which enables this feature by default.
-	//    https://cs.opensource.google/go/go/+/refs/tags/go1.18.1:src/cmd/go/internal/test/test.go;l=1331-1337
-	// 2. It provides a way to run hooks right before testing.M.Run() returns.
-	flag.Lookup("test.paniconexit0").Value.Set("true")
-
-	{{if ne .CoverMode ""}}
+{{if eq .CoverFormat "lcov"}}
+	panicOnExit0Flag := flag.Lookup("test.paniconexit0")
+	if panicOnExit0Flag != nil && panicOnExit0Flag.Value.String() != "false" {
```

Comment:
Can we check that it explicitly == "true"? Or maybe it would be better to do

```go
if panicOnExit0Flag != nil {
  testDeps.OrigionalPAnicOnExit = panicOnExit0Flag.Value.String()
}
```